src/space.js

```diff
@@ -271,6 +272,15 @@ export default class ColorSpace {
 
 		space = this.get(space);
 		this.registry[id] = space;
+
+		if (this.aliases) {
+			for (let alias of this.aliases) {
+				this.registry[alias] = space;
```

Comment:
A few things here: 
a) `this` is the `ColorSpace` instance here, so `this.registry` will be `undefined`
b) The constructor should be side effect free. It should be `register()` that adds stuff to `ColorSpace.registry`
c) The id for the `color()` function should be decoupled from aliases. In this case it happens that the alias is also an id for `color()`, but not sure this should be enforced by the API

Reply:
Oh, right, that's a good point, thank you! Sounds like space aliases need to be separate from color ID aliases 👍

Reply:
Also, what should be the expected behavior if one registers two spaces with the same id or alias? Right now each one will overwrite the previous one.

Reply:
Throwing a runtime error seems like the best bet to me. Having a unit test that checks to make sure there aren't any such errors on the full lib should then be enough to make sure the build doesn't break.

How does that sound?

Reply:
Not sure about a runtime error, users should be able to call `register()` multiple times from different parts of the code to register the same color space. Though perhaps we could only throw the error if the color space object is different.

Reply:
Yes, I totally agree that it shouldn't error out if registering the same space multiple times. The code should be doing that already, but do let me know if I missed something!